base/src/main/java/com/smartdevicelink/proxy/rpc/ShowConstantTbtResponse.java

```diff
@@ -44,23 +44,31 @@
  *
  * @since SmartDeviceLink 2.0
  */
-public class ShowConstantTbtResponse extends RPCResponse {
```

Comment:
@kostyaBoss I would revert these changes in `ShowConstantTbtResponse.java`.  This PR only deals with the `ShowConstantTbtRequest` so these changes are out of scope of the PR.